control/robust.py

```diff
@@ -356,8 +362,60 @@ def mixsyn(g,w1=None,w2=None,w3=None):
     """
     nmeas = g.outputs
     ncon = g.inputs
-    p = augw(g,w1,w2,w3)
+    p = augw(g, w1, w2, w3)
+
+    k, cl, gamma, rcond = hinfsyn(p, nmeas, ncon)
+    info = gamma, rcond
+    return k, cl, info
+
 
-    k,cl,gamma,rcond=hinfsyn(p,nmeas,ncon)
-    info = gamma,rcond
-    return k,cl,info
+def makeweight(dcgain, wc, hfgain, order=1, Ts=None):
+    """ Compute a weight transfer function, noted W
+
+    Parameters
+    ----------
+        dcgain : double positive scalar
+            Low frequency gain of 1/W; should be < 1
+        wc : double strictly positive scalar
+            Design frequency (where |W| is approximately 1)
+        hfgain : double positive scalar
+            High frequency gain of 1/W; should be > 1
+        order : int strictly positive scalar - optional
+            Order of the output transfer function
+        Ts : double strictly positive scalar - optional
```

Comment:
Shouldn't the sampling time come from the system (`sys.dt`)?  What happens if they are not compatible?